From b8eef4848f55efbc2657cc6e521d65eb48fe1d56 Mon Sep 17 00:00:00 2001
From: Vitaly Kuznetsov <vkuznets@redhat.com>
Date: Thu, 9 Oct 2014 12:42:38 -0400
Subject: [redhat] RHMAINTAINERS: Add Hyper-V as a supported subsystem

Message-id: <1412858558-4935-1-git-send-email-vkuznets@redhat.com>
Patchwork-id: 97299
O-Subject: [RHEL7 kernel PATCH] RHMAINTAINERS: Add Hyper-V as a supported subsystem
Bugzilla:
RH-Acked-by: Jason Wang <jasowang@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Jeff Nelson <jenelson@redhat.com>
RH-Acked-by: Ronen Hod <rhod@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

We support RHEL on Hyper-V for a long time already, make RHMAINTAINERS
reflect the reality.

Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 7c76008..bb71627 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -311,6 +311,22 @@ S: Maintained
 F: mm/memory-failure.c
 F: mm/hwpoison-inject.c
 
+Hyper-V CORE AND DRIVERS
+M: Jason Wang <jasowang@redhat.com>
+M: Vitaly Kuznetsov <vkuznets@redhat.com>
+S: Maintained
+F: arch/x86/include/asm/mshyperv.h
+F: arch/x86/include/uapi/asm/hyperv.h
+F: arch/x86/kernel/cpu/mshyperv.c
+F: drivers/hid/hid-hyperv.c
+F: drivers/hv/
+F: drivers/input/serio/hyperv-keyboard.c
+F: drivers/net/hyperv/
+F: drivers/scsi/storvsc_drv.c
+F: drivers/video/hyperv_fb.c
+F: include/linux/hyperv.h
+F: tools/hv/
+
 IBM IBMVETH ETHERNET DRIVER (ibmveth)
 M: Brad Peters <bpeters@redhat.com>
 S: Maintained
-- 
1.7.1